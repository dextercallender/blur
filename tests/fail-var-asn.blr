void main() {
    int a = "hi";
}