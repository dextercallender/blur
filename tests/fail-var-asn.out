Fatal error: exception Failure("variable declaration type mismatch")
